doc/manuals/devel/HttpHeaders.md

```diff
@@ -0,0 +1,354 @@
+List of HTTP Headers used by Orion
+----------------------------------------------------------------------------------------------------
+
+1)  Accept                  
+2)  Content-Length          
+3)  Content-Type
+4)  Connection                
+5)  Expect                                                                                
+6)  Host                        	                                                      
+7)  Origin                                                       
+8)  User-Agent              
+9)  X-Forwarded-For                                                               
+10) X-Real-IP
+
+11) Access-Control-Allow-Origin
+12) Access-Control-Allow-Headers                                                                              
+13) Access-Control-Allow-Methods
+14) Access-Control-Max-Age                                 
+15) Access-Control-Expose-Headers
+16) Allow                                                                                    
+17) Location                                                 
+18) Ngsiv2-AttrsFormat                                    
+19) Fiware-Service
+20) Fiware-Servicepath
+21) Fiware-Total-Count
+22) Fiware-Correlator                                     
+
+23) X-Auth-Token
+
+
+---------------------------------------------------------------------------------------------------------------------
+
+# HTTP Headers used by Orion with their Description:
+
+--------------------------------------------------------------------------------------------------------------------
+
+
+## 1. Accept  : 
+
+Accept HTTP Header is used in Request URL. 
+Accept HTTP Header is used with the request URL for accepting the response in well defined format. 
+For example	in request URL if we type:
+	
+		(e.g):      curl ... 'Accept: applicattion/json'
```

Comment:
I suggest change this for just

```
curl ... 'Accept: applicattion/json'
```

I mean, no need of so big indentation and the  `(e.g.)` is quite redundant. Please, apply the same to all similar cases in this file.